### PR TITLE
CSS module: css text

### PR DESCRIPTION
**Pull request by estelle**
Update CSS text module landing page.

### CODE REVIEW

**Review by Imran-imtiaz48 — COMMENTED** [review]
The CSS Text module page covers key aspects of text manipulation in CSS, detailing properties for text wrapping, alignment, spacing, and other formatting options. This page's clear layout and detailed references to individual properties are helpful for developers needing to understand and apply CSS text manipulation.

Suggested Improvements

1. Expand Property Descriptions:

Currently, the property list is a straightforward reference, but adding brief descriptions or usage examples for each property could significantly improve clarity and usability.

Consider including practical code snippets for essential properties like text-align, line-break, and white-space to illustrate common use cases.



2. Specification Links:

The inclusion of links to CSS Text Level 3 and CSS Text Level 4 specifications is helpful, but adding brief summaries of each specification’s goals and highlighting differences could aid users in choosing the appropriate properties for different browser compatibilities.



3. Browser Support Information:

Some properties, particularly experimental ones, may not be universally supported. Adding a browser compatibility chart (using resources like MDN data) would be beneficial, especially for properties labeled as experimental, like text-spacing-trim and text-size-adjust.



4. Expand on Non-Supported Properties:

Properties listed as unsupported, such as hyphenate-limit-last and text-group-align, could include notes on their potential applications once they become widely available. This helps developers anticipate future support and understand their purpose.



5. Additional Guides:

The “Wrapping and Breaking Text” guide is valuable, but additional guides for common challenges, such as vertical text alignment, handling text overflow gracefully, and managing word breaking for multiple languages, would enhance this section.




Overall Impression

This page is a solid foundation, providing a comprehensive reference for CSS text properties. Adding examples, browser compatibility details, and further guidance would transform it into a more user-friendly and thorough resource for developers.

File: files/en-us/web/css/css_text/index.md [hunk 1]
```diff
@@ -17,25 +16,69 @@ The **CSS text** module defines how to perform text manipulation, like line brea
...
 - {{cssxref("text-transform")}}
 - {{cssxref("text-wrap")}}
+- {{cssxref("text-wrap-mode")}}
+- {{cssxref("text-wrap-style")}}
 - {{cssxref("white-space")}}
 - {{cssxref("white-space-collapse")}}
 - {{cssxref("word-break")}}
 - {{cssxref("word-spacing")}}
 
+The specification also defines the `hyphenate-limit-last`, `hyphenate-limit-lines`, `hyphenate-limit-zone`, `line-padding`, `text-align-all`, `text-autospace`, `text-group-align`, `text-spacing`, `white-space-trim`, `word-space-transform`, `wrap-after`, `wrap-before`, and `wrap-inside` properties, which are not yet supported bay any browser.
```
Comment (bsmth) [inline files/en-us/web/css/css_text/index.md:40]:
```suggestion
The specification also defines the `hyphenate-limit-last`, `hyphenate-limit-lines`, `hyphenate-limit-zone`, `line-padding`, `text-align-all`, `text-autospace`, `text-group-align`, `text-spacing`, `white-space-trim`, `word-space-transform`, `wrap-after`, `wrap-before`, and `wrap-inside` properties, which are not yet supported by any browser.
```

File: files/en-us/web/css/css_text/index.md [hunk 2]
```diff
@@ -17,25 +16,69 @@ The **CSS text** module defines how to perform text manipulation, like line brea
 ### Properties
 
 - {{cssxref("hanging-punctuation")}}
+- {{cssxref("hyphenate-character")}}
 - {{cssxref("hyphenate-limit-chars")}}
 - {{cssxref("hyphens")}}
 - {{cssxref("letter-spacing")}}
 - {{cssxref("line-break")}}
-- {{cssxref("overflow-wrap")}}
+- {{cssxref("overflow-wrap")}} (and the {{cssxref("word-wrap")}} alias)
```
Comment (bsmth) [inline files/en-us/web/css/css_text/index.md:24]: This redirects automatically, so we can consider having it in backticks only:

```suggestion
- {{cssxref("overflow-wrap")}} (and the `word-wrap` alias)
```

**Comment by bsmth** [comment]
Thanks, Estelle, a couple of comments to consider, otherwise looking good!

File: files/en-us/web/css/css_text/index.md [hunk 3]
```diff
@@ -17,25 +16,69 @@ The **CSS text** module defines how to perform text manipulation, like line brea
...
 
 {{Specifications}}
+
+## See also
+
+- [CSS writing modes](/en-US/docs/Web/CSS/CSS_writing_modes) module
+- [CSS overflow](/en-US/docs/Web/CSS/CSS_overflow) module
+- [CSS fonts](/en-US/docs/Web/CSS/CSS_fonts) module
+- [CSS ruby layout](/en-US/docs/Web/CSS/CSS_ruby_layout) module
+- [CSS inline](/en-US/docs/Web/CSS/CSS_inline) module
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/css_text/index.md:84]: working on that page today.

File: files/en-us/web/css/css_text/index.md [hunk 4]
```diff
@@ -17,25 +16,69 @@ The **CSS text** module defines how to perform text manipulation, like line brea
...
```
Comment (estelle) [inline files/en-us/web/css/css_text/index.md:84]:
```suggestion
```

File: files/en-us/web/css/css_text/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/css_text/index.md]: Not adding that module yet (only CSS2.1 features have been implemented), so removed.

[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/css/css_text/index.md]: OK ty 👍🏻

**Review by bsmth — APPROVED** [review]
LGTM, thank you!

**Pull request merged**